src/esn.inbox.libs/app/app.module.js

```diff
@@ -0,0 +1,9 @@
+
+'use strict';
+
+angular.module('esn.inbox.libs', [
+   
```

Comment:
you're missing dependencies here. The service [inboxRestangular](https://github.com/OpenPaaS-Suite/esn-frontend-inbox/blob/1251780f2843670867ede750d8546666e59401e6/src/esn.inbox.libs/app/services/inbox-restangular.service.js#L7) uses `Restangular` and `httpErrorHandler`, you should:
- for `restangular`, add the angular dependency only, as restangular is globally required in the [vendor-libs](https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/main/src/frontend/vendor-libs.js#L59)
- for `httpErrorHandler`, it's a service from `esn.http`, so you have to add it as an angular dependency in this array, and also you need to `require` it at the end of this file